concent_api/conductor/service.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from django.db.models import Q
+
+from common.constants import ErrorCode
+from conductor.exceptions import VerificationRequestAlreadyInitiatedError
+from conductor.models import ResultTransferRequest
+from conductor.models import UploadReport
+from conductor.models import VerificationRequest
+from core.tasks import result_upload_finished
+
+
+logger = logging.getLogger(__name__)
+
+
+def report_finished_upload(file_path: str, result_transfer_request: ResultTransferRequest) -> None:
```

Comment:
can this be called with `result_transfer_request` where `result_transfer_request.upload_finished = True`?

Reply:
Yes, this object may exists after `get-task-results` use case was finished, and we are in `verification` use case, and Conductor's `report_upload` endpoint is called.